README.md

```diff
@@ -40,7 +40,14 @@ sudo snap install --edge nvim --classic
 
 [Other methods of installation are available in the official repository of Neovim](https://github.com/neovim/neovim/wiki/Installing-Neovim)
 
-Instructions for **NodeJS** are also available in [the official repo.](https://github.com/nodesource/distributions/blob/master/README.md)
+
+To install **NodeJS** in the most convinient way, install NodeJS via NVM (Node Version Manager).
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
+# restart terminal
+nvm install --lts
+```
+> However, **NodeJS** is available on the [official website](https://nodejs.org/en/).
```

Comment:
Why solely link to the site is more efficient in comparison with direct link to the installation guide from official repo?

Reply:
[NodeSource](https://github.com/nodesource/distributions/blob/master/README.md) is NOT the official repo.